radio/src/gui/colorlcd/model_setup.cpp

```diff
@@ -641,15 +723,27 @@ class ModuleWindow : public FormGroup {
 
         // Multi type (CUSTOM, brand A, brand B,...)
         int multiRfProto = g_model.moduleData[moduleIdx].getMultiProtocol();
-				
+
+        // initialize rfProtocols if it is not initialized already.  This is a hack
+        // because i dont know how to construct it at the exact right time after the module
+        // is loaded
+        if (rfProtocols == nullptr)
+            rfProtocols = new RFProtocols(STR_MULTI_PROTOCOLS, MODULE_SUBTYPE_MULTI_LAST);
```

Comment:
Can you double check what's going on here... At present, the Config entry (83) is being shown at the end of the list, and if it is selected it will freeze the radio completely, requiring the battery disconnected. If run on `simu` it gets me
```
15.85s: RFProtocols::getProtoStringLength()
terminate called after throwing an instance of 'std::out_of_range'
  what():  vector::_M_range_check: __n (which is 83) >= this->size() (which is 83)
```
Adding +1 to MODULE_SUBTYPE_MULTI_LAST at this line fixes it in `simu`, and the Config entry is shown, but I'm off to bed now, so haven't tested if it has broken anything else, or is the correct fix.

Reply:
Crash is gone now :)